src/ocamlorg_frontend/pages/community.eml

```diff
@@ -9,198 +9,343 @@ Community_layout.single_column_layout
+        <li class="flex mb-3">
+          <%s! Icons.email "h-10 mr-12 hidden md:block text-primary dark:text-dark-primary" %>
+         <div class="flex flex-col">
+          <div class="flex items-center mb-2 md:mb-0">
+            <%s! Icons.email "h-10 mr-4 md:hidden block text-primary dark:text-dark-primary" %>
+            <p class="text-lg text-title dark:text-dark-title font-bold">Caml User's Mailing List</p>
+          </div>
+          <p class="text-content dark:text-dark-content"> Share experience, exchange ideas and code, and report on applications - <a href="https://inbox.vuxu.org/caml-list/" class="text-primary dark:text-dark-primary hover:underline">inbox.vuxu.org/caml-list/</a></p>
+         </div>
+        </li>
+      </ul>
   </div>
 </div>
-<% if List.length upcoming_events > 0 then ( %>
-    <div class="w-full">
-      <div class="container-fluid pt-6 pb-12">
-          <p class="uppercase text-sm text-content dark:text-dark-content tracking-widest font-medium mb-2">events</p>
-          <h2 class="font-bold text-2xl text-title dark:text-dark-title mb-2">Upcoming Events</h2>
-          <div class="grid lg:grid-cols-3 md:grid-cols-2 gap-8">
-              <% upcoming_events |> List.iter (fun (event : Data.Event.t) -> %>
-              <%s! Cards.Event.upcoming_event event %>
-              <% ); %>
-          </div>
-      </div>
-    </div>
-<% ); %>
 <%s! Cards.Event.script %>
-<div class="w-full border-b">
+<div id="events" class="w-full bg-sand dark:dark-section-strong-contrast border border-separator_20">
   <div class="container-fluid pt-6 pb-12">
-    <p class="uppercase text-sm text-content dark:text-dark-content tracking-widest font-medium mb-2">workshop</p>
-    <h2 class="font-bold text-2xl text-title dark:text-dark-title mb-2">Workshops we have had up until now</h2>
-    <% (match upcoming_workshops with
-        | [] -> ()
-        | _ -> %>
-    <div class="text-center text-white text-xl font-bold">
-        Upcoming Workshops
-    </div>
+      <% let (upcoming_events, event_types) = events in %>
+      <p class="uppercase text-sm text-content dark:text-dark-content tracking-widest font-medium my-4">Events</p>
+      <% let event_num = match List.length upcoming_events with 0 -> "No Event" | 1 -> "Upcoming Events (1)" | l -> Printf.sprintf "Upcoming Events (%i)" l in %>
+      <h5 class="font-bold text-title dark:text-dark-white mb-4">
+          <%s event_num %>
+       </h5> 
+        <div class="h-14 w-64 mt-2 border-primary dark:border-primary mb-4">
+          <form action="<%s Url.community %>" method="GET">
+                <label for="e" class="sr-only">Location</label>
+                <%s! Forms.select
+                  ~attrs:{|id="e" name="e" onchange="this.form.submit()"|}
+                  ~options:([("All", "Event Types")] @ ( event_types |> List.map (fun item -> (item, item))))
+                  ~selected:selected_event
+                 "" %>
+          </form>
+       </div> 
```

Comment:
filtering on the community page doesn't make much sense, I would expect this to redirect to the filtered events page when people select an event type to filter by (but we can merge this as is and make a PR to change it afterwards, when completing the events page)